### ISSUE TITLE
CredScan failure

### ISSUE THREAD

**Issue by jeffschwMSFT** [dotnet/dnceng#2921]
### Build

https://dev.azure.com/dnceng/internal/_build/results?buildId=2456692

```
##[error]ERROR: ld.so: object '/__w/_temp/codeql3000/distribution/codeql/tools/linux64/${LIB}_${PLATFORM}_trace.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored. - 
```

### Known issue core information

Fill out the known issue JSON section by following the [step by step documentation on how to create a known issue](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssueJsonStepByStep.md#how-to-create-a-known-issue-step-by-step)

```json
 {
    "ErrorMessage" : "ERROR: ld.so: object '/__w/_temp/codeql3000/distribution/codeql/tools/linux64/${LIB}_${PLATFORM}_trace.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file)",
    "BuildRetry": false,
    "ErrorPattern": "",
    "ExcludeConsoleLog": false
 }
 ```

 @dotnet/dnceng

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements

**Comment by markwilkie** [comment]
I presume this is a 1ES PT issue?

**Comment by jeffschwMSFT** [comment]
indeed, this is the tracker so that we see how often it is failing.  I am in the process of finding the right owner - do you have a contact?

**Comment by markwilkie** [comment]
Who are the right people to contact about 1ESPT stuff again @missymessa ?

**Comment by missymessa** [comment]
https://eng.ms/docs/microsoft-security/microsoft-threat-protection-mtp/cloud-and-enterprise-security-cesec/security-integration/guardian-wiki/microsoft-guardian/general/guardian-tools

**Comment by jeffschwMSFT** [comment]
thanks, issue submitted/email sent.
